src/dialect/postgresql.rs

```diff
@@ -20,6 +21,23 @@ use crate::tokenizer::Token;
 #[derive(Debug)]
 pub struct PostgreSqlDialect {}
 
+const DOUBLE_COLON_PREC: u8 = 140;
+const BRACKET_PREC: u8 = 130;
+const COLLATE_PREC: u8 = 120;
+const AT_TZ_PREC: u8 = 110;
+const CARET_PREC: u8 = 100;
+const MUL_DIV_MOD_OP_PREC: u8 = 90;
+const PLUS_MINUS_PREC: u8 = 80;
+// there's no XOR operator in PostgreSQL, but support it here to avoid breaking tests
+const XOR_PREC: u8 = 75;
```

Comment:
I'm not sure what to do about this, if we remove `XOR` logic from Postgres, one tests fails, but I guess people might well be using it.

Reply:
yeah I think we should just leave it in unless there is a compelling reason to take it out